Sources/HaCWebsiteLib/ViewModels/LandingPage/EventFeature.swift

```diff
@@ -31,6 +31,10 @@ struct EventFeature: LandingFeature {
       return El.Div[Attr.className => "EventFeature__date EventFeature__date--live"].containing(
         "On now"
       )
+    } else if isToday {
+      return El.Div[Attr.className => "EventFeature__date EventFeature__date--live"].containing(
```

Comment:
Now that the `EventFeature__date--live` class applies to more than just when the event is 'live' (i.e. happening right now), perhaps we should rename it (e.g. to `EventFeature__date--highlight`)

Reply:
Sure that makes sense. So should I add the highlight class only for the "Today" label?

Reply:
Since both "Today" and "On now" should be highlighted, let's use the highlight class for both and change the CSS accordingly